### PR TITLE
[DE] Add `Music theory/Rhythm`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Music_theory/Rhythm/de.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Rhythmus
+
+**Rhythmus** ist "die Anordnung von Tönen, Stille und Betonung in einem Song".[^rhythm-ref] Beim Erstellen von [Beatmaps](/wiki/Beatmap) achten Mapper darauf, wo Töne in der Musik erscheinen, damit sie wissen, wo [Hit-Objekte](/wiki/Gameplay/Hit_object) und [Hitsounds](/wiki/Beatmapping/Hitsound) platziert werden sollen, um dem Rhythmus zu folgen und bestimmte Teile davon zu betonen. Als Rhythmusspiel ist Rhythmus ein fester Bestandteil jeder Beatmap in osu!.
```
Comment (FreezyLemon) [inline wiki/Music_theory/Rhythm/de.md:7]: `"die Anordnung von Tönen, Stille und Betonung in einem Song"`
Should probably mention it's translated from the original English version (maybe in the footnote).

Comment (FreezyLemon) [inline wiki/Music_theory/Rhythm/de.md:7]: The wording around `Rhythmusspiel` is weird. Let's keep it simple:

```suggestion
**Rhythmus** ist "die Anordnung von Tönen, Stille und Betonung in einem Song".[^rhythm-ref] Beim Erstellen von [Beatmaps](/wiki/Beatmap) achten Mapper darauf, wo Töne in der Musik erscheinen, damit sie wissen, wo [Hit-Objekte](/wiki/Gameplay/Hit_object) und [Hitsounds](/wiki/Beatmapping/Hitsound) platziert werden sollen, um dem Rhythmus zu folgen und bestimmte Teile davon zu betonen. Da osu! ein Rhythmusspiel ist, ist Rhythmus ein fester Bestandteil jeder Beatmap.
```

Comment (FreezyLemon) [inline wiki/Music_theory/Rhythm/de.md:7]: Let's try and remove one `Nebensatz` to make it less complex:

```suggestion
**Rhythmus** ist "die Anordnung von Tönen, Stille und Betonung in einem Song".[^rhythm-ref] Beim Erstellen von [Beatmaps](/wiki/Beatmap) achten Mapper darauf, wo Töne in der Musik erscheinen, um [Hit-Objekte](/wiki/Gameplay/Hit_object) und [Hitsounds](/wiki/Beatmapping/Hitsound) so platzieren zu können, dass sie dem Rhythmus folgen und bestimmte Teile davon betonen. Als Rhythmusspiel ist Rhythmus ein fester Bestandteil jeder Beatmap in osu!.
```

**Review by FreezyLemon — CHANGES_REQUESTED** [review]

File: wiki/Music_theory/Rhythm/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Music_theory/Rhythm/de.md:7]: Not really needed in my opinion as the title already lets the reader assume the article is in English. If you still prefer to have it, I would write something like this (based on the [rule for citations](https://osu.ppy.sh/wiki/de/Article_styling_criteria/Writing#blockzitate))

```md
[^rhythm-ref]: ["Understanding Rhythm in Music: 7 Elements of Rhythm" auf MasterClass](https://www.masterclass.com/articles/understanding-rhythm-in-music) (aus dem Englischen übersetzt)
```

Comment (FreezyLemon) [reply wiki/Music_theory/Rhythm/de.md:7]: I would prefer it, yes. It seems like good sense to include a note like this somewhere, as the translation is an interpretation on your part, which means it's not technically a direct quote anymore.

**Review by FreezyLemon — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**